app/src/pages/playground/PlaygroundDatasetSection.tsx

```diff
@@ -47,21 +48,30 @@ export function PlaygroundDatasetSection({ datasetId }: { datasetId: string }) {
               </Text>
             ) : null}
           </Flex>
-          {experimentId != null && (
-            <Button
-              size={"compact"}
-              variant="default"
-              disabled={isRunning}
-              icon={<Icon svg={<Icons.ExperimentOutline />} />}
-              onClick={() => {
-                navigate(
-                  `/datasets/${datasetId}/compare?experimentId=${experimentId}`
-                );
-              }}
-            >
-              View Experiment
-            </Button>
-          )}
+          <Flex gap={"size-100"} alignItems={"center"}>
+            {isRunning && (
+              <Text color="text-700">
+                Run in progress, navigating away from the page could result in
```

Comment:
The second part of this message feels redundant. I'd personally rather let the user discover our lack of job based runs if they try to leave first 😅

Reply:
Yeah i think that makes ense, the loader on the button probably should alert them to that so maybe this text is unnecessary. But i'll leave Run in progress for now